src/types/chart/common_bar_combo.ts

```diff
@@ -6,6 +6,8 @@ export interface ComboBarChartDefinition {
   readonly dataSetsHaveTitle: boolean;
   readonly labelRange?: string;
   readonly title: string;
+  readonly isTitleCellReferenced?: boolean;
+  readonly titleCellReference?: string;
```

Comment:
I think there's a design problem here. I don't think we should have both `title` and `titleCellReference`. In the best case that's duplicated information, in the worth case it can very easily lead to issue beacuse of unhandled cases (like in you code).

There's a big missing part in your code. You keep using chart.title to generate the runtime of the chart. But chart.title is equal to the referencing cell content only when updating the title in the panel. Afterwards, when the cell's content changes, you don't update the title in the runtime with the new value.

I think this would have been easier to spot if we only had a single option `title` , which would be either a reference of a raw string.